test/conftest.py

```diff
@@ -543,7 +543,7 @@ def output_lockstep(self) -> Iterator[threading.Barrier]:
         self.put_outputs_barrier = threading.Barrier(1)
 
     async def FunctionGetInputs(self, stream):
-        self.get_inputs_barrier.wait()
+        await asyncio.get_running_loop().run_in_executor(None, self.get_inputs_barrier.wait)
```

Comment:
This was mostly a code sanity thing - I realized this could block the test server event loop otherwise if you have a malformed test using input_lockstep